policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credit.py

```diff
@@ -20,11 +20,10 @@ def formula(tax_unit, period, parameters):
         has_not_taken_lump_sum_distribution = person(
             "oh_has_not_taken_oh_lump_sum_credits", period
         )
-        age = person("age", period)
 
-        any_elderly = tax_unit.any(age >= p.age_threshold)
+        age = person("age", period)
+        elderly = age >= p.age_threshold
+        eligible = tax_unit.sum(has_not_taken_lump_sum_distribution & elderly)
```

Comment:
Please select the correct one
```suggestion
        eligible = has_not_taken_lump_sum_distribution & elderly
        count_eligible = tax_unit.sum(eligible)
        credit_amount_per_person = p.agi_limit.calc(tax_unit("oh_agi", period))
        return count_eligible * credit_amount_per_person
```
```suggestion
        eligible = has_not_taken_lump_sum_distribution & elderly
        eligible_filer = tax_unit.any(eligible)
        credit_amount = p.agi_limit.calc(tax_unit("oh_agi", period))
        return eligible_filer * credit_amount
```